nexus/lib/pwscf_analyzer.py

```diff
@@ -241,6 +236,15 @@ def analyze(self):
             read_rel      = False
             for i in xrange(len(lines)):
                 l = lines[i]
+                if 'End of self-consistent calculation' in l:
```

Comment:
What about the non-self consistent case?  Shouldn't we also keep the original empty initialization of bands, etc?

Reply:
Good catch. I have restored the original inits in the most recent commit.

Reply:
Additionally, I added a condition to prevent the inits from being performed redundantly for the first iteration of scf convergences.